packages/smart-wallet/src/smartWallet.js

```diff
@@ -490,35 +481,20 @@ const SmartWalletKit = defineVirtualFarClassKit(
   },
   {
     finish: ({ state, facets }) => {
-      const { invitationBrand, invitationIssuer, invitationPurse, bank } =
-        state;
+      const invitationDisplayInfo = { assetKind: AssetKind.SET }; // XXX
```

Comment:
what should this be instead?

maybe a Zoe const?

Reply:
A Zoe const might work... but I went with the caller passing it in along with the brand and issuer.